lib/matplotlib/font_manager.py

```diff
@@ -128,8 +128,8 @@
     "/Network/Library/Fonts/",
     "/System/Library/Fonts/",
     # fonts installed via MacPorts
-    "/opt/local/share/fonts"
-    ""
+    "/opt/local/share/fonts",
```

Comment:
This comma changed semantics from implicit string concatenation to a new empty string in the list.

Reply:
This is correcting a bug, we want to pick up fonts in the current directory.

Reply:
Only on OSX, not windows or Linux?
Doesn't seem like something we'd want.

Reply:
Hmm, that seems like a bug that it is missing in the other cases.

On the other hand, this recursion issue seems to be causing grief for many people and as this is cached,  using the cwd is also a bit off (as the next time you read the cache your cwd is different).

On balance, I think my initial reaction to this was wrong.